sssom/parsers.py

```diff
@@ -499,6 +518,144 @@ def from_obographs(
+    - refSetId,SctId,Set to one of the children of the |Complex map type| concept in the metadata hierarchy.,The only
+    value in the entire set is '5991000124107', which has label 'ICD-10-CM complex map reference set' (
+    https://www.findacode.com/snomed/6011000124106--icd-10-cm-complex-map-reference-set.html).
+    - referencedComponentId,SctId,The SNOMED CT source concept ID that is the subject of the map record.,
+    - mapGroup,Integer,An integer identifying a grouping of complex map records which will designate one map target at
+    the time of map rule evaluation. Source concepts that require two map targets for classification will have two sets
+    of map groups.,
+    - mapPriority,Integer,Within a map group, the mapPriority specifies the order in which complex map records should be
+    evaluated to determine the correct map target.,
+    - mapRule,String,A machine-readable rule, (evaluating to either ‘true’ or ‘false’ at run-time) that indicates
+    whether this map record should be selected within its map group.,
+    - mapAdvice,String,Human-readable advice that may be employed by the software vendor to give an end-user advice on
+    selection of the appropriate target code. This includes a) a summary statement of the map rule logic, b) a statement
+    of any limitations of the map record and c) additional classification guidance for the coding professional.,
+    - mapTarget,String,The target ICD-10 classification code of the map record.,
+    - correlationId,SctId,A child of |Map correlation value| in the metadata hierarchy, identifying the correlation
+    between the SNOMED CT concept and the target code.,
+    - mapCategoryId,SctId,Identifies the SNOMED CT concept in the metadata hierarchy which is the MapCategory for the
+    associated map record. This is a subtype of 447634004 |ICD-10 Map Category value|.,
+    """
+    # https://www.findacode.com/snomed/447561005--snomed-ct-source-code-to-target-map-correlation-not-specified.html
+    match_type_snomed_unspecified_id = 447561005
+    prefix_map = _ensure_prefix_map(prefix_map)
+    ms = _init_mapping_set(meta)
+
+    mlist: List[Mapping] = []
+    for _, row in df.iterrows():
+        mdict = {
+            'subject_id': f'SNOMED:{row["referencedComponentId"]}',
+            'subject_label': row['referencedComponentName'],
+
+            # 'predicate_id': 'skos:exactMatch',
+            # - mapCategoryId: can use for mapping predicate? Or is correlationId more suitable?
+            #   or is there a SKOS predicate I can map to in case where predicate is unknown? I think most of these
+            #   mappings are attempts at exact matches, but I can't be sure (at least not without using these fields
+            #   to determine: mapGroup, mapPriority, mapRule, mapAdvice).
+            # mapCategoryId,mapCategoryName: Only these in set: 447637006 "MAP SOURCE CONCEPT IS PROPERLY CLASSIFIED",
```

Comment:
### `mapCategoryId` and `mapCategoryName`
I'm not sure if there's any useful information here that could be put anywhere than the `other` field. There are only 3 values that show up in the entire mapping dataset for both `mapCategoryId` and `mapCategoryName`, and I listed them out int he comment here.

Reply:
@matentzn